Updates to How SSO Works article (#3093)
FusionAuth/fusionauth-site: astro/src/content/articles/authentication/how-sso-works.mdx

Why? Because your organization typically has more than one customer facing application. Even if you start with a custom application, SaaS tools such as support forums, ticketing systems, or chat require your customers to log in. SSO can minimize how many times a customer has to log in.

## What Does SSO Mean?

SSO is an abbreviation for "single sign-on." Integrating single sign-on into your applications means your users can access all their applications with one set of credentials. Those could be a username and password, a magic link, or a WebAuthn passkey.

Offering this allows you to have one view of the user, which is great for security and data consistency.

Single sign-on also allows users to minimize the time they spend getting into your application and maximize the time spent using it.

### What Is An SSO Token?

> Think of an SSO token as a digital key that contains user-identifying information, such as a username or email address. When a user logs in through an SSO service, this token is created and stored either in the user's browser or within the SSO service's servers.

Reviewer: ```suggestion
Think of an SSO token as data that contains user-identifying information, such as a username or email address. When a user logs in through an SSO service, this token is created and stored either in the user's browser or within the SSO service's servers.
```

> The token is then passed between systems during the SSO process to verify the user's identity. This means that once a user is authenticated, they can seamlessly access other connected applications without having to log in again. The token is typically  digitally signed to ensure it comes from a trusted source, adding an extra layer of security to the authentication proces

Reviewer: ```suggestion
The token is then passed between systems during the SSO process to verify the user's identity. This means that once a user is authenticated, they can seamlessly access other connected applications without having to log in again. The token is typically  digitally signed to ensure it comes from a trusted source, adding an extra layer of security to the authentication process.
```

## What Are the Benefits of SSO?

We've already uncovered some of these, such as an improved user experience. But let's dive in deep with each of the benefits to see how they'll impact your customer.

### Enhanced Security

> SSO reduces the risk of password-related security breaches by eliminating the need for users to remember and manage multiple credentials across different applications. It centralizes authentication, making phishing attacks less effective and minimizing the overall attack surface for cybercriminals

Reviewer: ```suggestion
SSO reduces the risk of password-related security breaches by eliminating the need for users to remember and manage multiple credentials across different applications. It centralizes authentication, making phishing attacks less effective and minimizing the overall attack surface for cybercriminals.
```

### Streamlined User Experience

With SSO, users only need to authenticate once to access multiple applications. This eliminates the need for repeated logins, reducing the brain burden associated with managing credentials. This streamlined experience can not only improve user satisfaction, your users might find greater productivity too.

### Simplified Access Management

> SSO solutions provide your team with better visibility and control over user access. Admins can manage user accounts, permissions, and access controls, making it easier to provision and remove users across mutliple systems.

Reviewer: ```suggestion
SSO solutions provide your team with better visibility and control over user access. Admins can manage user accounts, permissions, and access controls, from a single source making it easier to provision and remove users across multiple systems.
```

### Regulatory Compliance

> Centralized authentication and access controls in SSO help organizations meet regulatory requirements. When data access, user authentication, and audit trails become table stakes, then HIPAA, SOC2, and other standards are easier to manage.

Reviewer: "When data access....become table stakes....".  I think this needs to be reworded as those are always "table stakes" in my mind and should be for anyone.  This kind makes it seem like there are some cases where FusionAuth doesn't care about that. IMHO

> Centralized authentication and access controls in SSO help organizations meet regulatory requirements. Data access, user authentication, and audit trails are table stakes for FusionAuth. This makes HIPAA, SOC2, and other standards are easier to manage.

Reviewer: ```suggestion
Centralized authentication and access controls in SSO help organizations meet regulatory requirements. Data access, user authentication, and audit trails are table stakes for FusionAuth. This makes HIPAA, SOC2, and other standards easier to manage.
```

### Improved Productivity

Let's say that you're a business, creating an app that your employees will use to log in. As time goes on and you integrate more systems. allowing SSO lets people focus on their tasks rather than managing credentials.

## Is SSO Secure?

> Absolutely! As long as you know what you're doing. SSO can be highly secure, but it depends on how it's implemented and managed. Let's break it down:

Reviewer: ```suggestion
Absolutely! SSO can be highly secure, but it needs to be implemented and managed correctly. Let's break it down:
```

### Centralized Security

> SSO reduces the number of attack surfaces because users only login once, and with only a single set of credentials.

Reviewer: I have always had concerns that it goes the other way too.  If they get a hold of one, then they have all. :( but the idea is there.  When talking about security, you may say something like users can be disabled or removed in one central source for all applications.  No forgetting to remove them from a rarely used app.

### Reduced Password Fatigue

Users only need to remember one complex password instead of many. They're then less likely to reuse passwords, or write them down.

### Multi-Factor Authentication (MFA)

> Most SSO solutions all for [MFA](/articles/authentication/multi-factor-authentication) as an extra layer of security. So even if a password is compromised, the bad actor still needs a second form of verification.

Reviewer: ```suggestion
Most SSO solutions offer [MFA](/articles/authentication/multi-factor-authentication) as an extra layer of security. So even if a password is compromised, the bad actor still needs a second form of verification.
```
I assume this is what you meant.
